src/pages/contact/index.tsx

```diff
@@ -1,27 +1,51 @@
 import React from "react";
 import Head from "next/head";
+import classnames from "classnames";
 
 import { ContactUsForm } from "../../components/ContactUsForm";
+import { PageBanner } from "../../components/Banner";
+import { DemoCourseCTA } from "../../components/DemoCourse";
+import { PageHeading } from "../../components/PageHeading";
+
+import styles from "./index.module.scss";
 
 export default function Contact() {
   return (
     <React.Fragment>
       <Head>
         <title>Contact us | PrairieLearn</title>
       </Head>
-      <div className="container my-5">
-        <h1 className="display-3">Contact us</h1>
-        <p className="lead mb-5">
-          Want to learn more about how PrairieLearn can help your course or
-          institution? Need to schedule a demo? Interested in managed hosting?
-          Let us know below and someone from our team will be in touch soon.
-        </p>
-        <div className="row justify-content-center">
+      <PageBanner
+        title="Ready to get started?"
+        subtitle="Request your course space here"
+      />
+
+      <div className={classnames("container-fluid py-4", styles.container)}>
+        <div className="container-md">
+          <div className="row">
+            <PageHeading>Contact us!</PageHeading>
+          </div>
+          <div className="row">
+            <div className="col-md-12">
+              Need to schedule a demo? Want to have access to your own course
+              space? Let us know below and someone from our team will be in
+              touch soon.
+            </div>
+          </div>
+        </div>
+
+        <div className="row justify-content-center my-5">
```

Comment:
This form appears realllllly wide:

<img width="1681" alt="Screen Shot 2022-01-28 at 1 12 06 PM" src="https://user-images.githubusercontent.com/1476544/151622105-873ce150-ee97-4a80-9e0e-4e2966732fca.png">

Maybe that was intentional, but I prefer the old, narrower width:

<img width="1667" alt="Screen Shot 2022-01-28 at 1 12 52 PM" src="https://user-images.githubusercontent.com/1476544/151622172-d8acb3eb-e192-4d40-b05e-362fb6ad6dde.png">